src/main/kotlin/com/team4099/robot2023/commands/led/LedCommand.kt

```diff
@@ -0,0 +1,43 @@
+package com.team4099.robot2023.commands.led
+
+import com.team4099.robot2023.Robot
+import com.team4099.robot2023.config.constants.LedConstants
+import com.team4099.robot2023.subsystems.drivetrain.drive.Drivetrain
+import com.team4099.robot2023.subsystems.led.Led
+import edu.wpi.first.wpilibj2.command.CommandBase
+import org.team4099.lib.units.derived.inDegrees
+import java.lang.Math.abs
+
+class LedCommand(val led: Led, val drivetrain: Drivetrain) : CommandBase() {
+
+  init {
+    addRequirements(led, drivetrain)
+  }
+
+  override fun execute() {
+    /* led.state =
+        when (Robot.isDisabled) {
+          LedConstants.LEDMode.DISABLED
+        }
+    */
+
+    if (kotlin.math.abs(drivetrain.gyroInputs.gyroPitch.inDegrees) > 2.5) {
+      if (drivetrain.gyroInputs.gyroPitch.inDegrees > 2.5) {
+        led.state = LedConstants.LEDMode.POS_LEVELED
+      } else {
+        led.state = LedConstants.LEDMode.NEG_LEVELED
+      }
```

Comment:
add a comment to switch from pitch to roll when we actually test the robot

Reply:
autobalance command currently uses pitch so i think this is fine, unless center of gravity ends up being more centered on x-axis(roll axis) and we change autobalance to use roll instead